src/apis/setting/menu.api.ts

```diff
@@ -0,0 +1,59 @@
+import { SetMenuData, SetOptionsData } from '@/types';
+import { axiosClient } from '../axios';
+
+export const getCategories = async () => {
+  const response = await axiosClient.get('/categories');
+
+  return response.data;
+};
+
+export const addCategories = async (categoryName: string) => {
+  const response = await axiosClient.post('/categories', { categoryName: categoryName });
```

Comment:
key와 value가 같을 때는 단축해서 표현해줄 수 있습니다~
```suggestion
  const response = await axiosClient.post('/categories', { categoryName });
```